YifanLiao.md

```diff
@@ -2,12 +2,15 @@ Last Week
 
 * [TV]
   - Write tv animation article
-  	- make 'Menu Group' component with Web Animation API
+  	- make 'Menu Group' component with Web Animation API http://codepen.io/begeeben/pen/ZGMZPX
+  	- finished
   - study web animation
+  - [WIP] [Bug 1144540 - [Stingray][Component] Unify package format for smart-button with other gaia-components]
+  	- r? & r-
+  - try to make browser work
```

Comment:
Please pay more effort on this, we definitely need it in the near future. Thanks.